backend/src/Designer/Controllers/Organisation/OrgTextController.cs

```diff
@@ -0,0 +1,112 @@
+    /// Initializes a new instance of the <see cref="OrgTextController"/> class.
+    /// </summary>
+    /// <param name="orgTextsService">The texts service.</param>
+    public OrgTextController(IOrgTextsService orgTextsService)
+    {
+        _orgTextsService = orgTextsService;
+    }
+
+    /// <summary>
+    /// Returns a JSON resource file for the given language code
+    /// </summary>
+    /// <param name="org">Unique identifier of the organisation responsible for the app.</param>
+    /// <param name="languageCode">The resource language id (for example <code>nb, en</code>)</param>
+    /// <returns>The JSON config</returns>
+    [HttpGet]
+    [Route("language/{languageCode}")]
+    public async Task<ActionResult<TextResource>> GetResource(string org, string languageCode)
+    {
+        try
+        {
+            string developer = AuthenticationHelper.GetDeveloperUserName(HttpContext);
+            TextResource textResource = await _orgTextsService.GetText(org, Repo, developer, languageCode);
+            return Ok(textResource);
+        }
+        catch (NotFoundException)
+        {
+            return NotFound($"Text resource, resource.{languageCode}.json, could not be found.");
+        }
+
+    }
+
+    /// <summary>
+    /// Save a resource file
+    /// </summary>
+    /// <param name="jsonData">The JSON Data</param>
+    /// <param name="languageCode">The resource language id (for example <code>nb, en</code> )</param>
+    /// <param name="org">Unique identifier of the organisation responsible for the app.</param>
+    /// <returns>The updated resource file</returns>
+    [HttpPost]
+    [Route("language/{languageCode}")]
+    public async Task<ActionResult<TextResource>> CreateResource([FromBody] TextResource jsonData, string languageCode, string org)
+    {
+        try
+        {
+            string developer = AuthenticationHelper.GetDeveloperUserName(HttpContext);
+            await _orgTextsService.SaveText(org, Repo, developer, jsonData, languageCode);
+
+            TextResource textResource = await _orgTextsService.GetText(org, Repo, developer, languageCode);
+            return Ok(textResource);
+        }
+        catch (ArgumentException e)
+        {
+            return BadRequest(e.Message);
+        }
+    }
+
+    /// <summary>
+    /// Method to update multiple texts for given keys and a given
+    /// language in the text resource files in the old format.
+    /// Non-existing keys will be added.
+    /// </summary>
+    /// <param name="org">Unique identifier of the organisation responsible for the app.</param>
+    /// <param name="keysTexts">List of Key/Value pairs that should be updated or added if not present.</param>
+    /// <param name="languageCode">The languageCode for the text resource file that is being edited.</param>
+    /// <remarks>Temporary method that should live until old text format is replaced by the new.</remarks>
+    /// <returns>The updated resource file</returns>
```

Comment:
A couple more `old format`s 🙂 
```suggestion
    /// <summary>
    /// Method to update multiple texts for given keys and a given
    /// language in the text resource files.
    /// Non-existing keys will be added.
    /// </summary>
    /// <param name="org">Unique identifier of the organisation responsible for the app.</param>
    /// <param name="keysTexts">List of Key/Value pairs that should be updated or added if not present.</param>
    /// <param name="languageCode">The languageCode for the text resource file that is being edited.</param>
    /// <returns>The updated resource file</returns>
```